### DISCUSSION TITLE
ImageViewer没有点击事件，那怎么让它关闭呢

### DISCUSSION THREAD

**Discussion by xixixinmu** [ant-design/ant-design-mobile#6272] [Q&A]
用了ImageViewer，然后是在Image上绑定的打开ImageViewer事件，但是在模拟器里蒙层上会有tabbar的返回箭头和胶囊，点箭头会返回上一页，点蒙层也没办法关闭ImageViewer，想问问是怎么回事。

**Comment by miracles1919** [comment]
来个复现看看，你可以通过 [codesandbox](https://codesandbox.io/s/antd-mobile-snrxr?file=/package.json) 或 [stackblitz](https://stackblitz.com/edit/antd-mobile?file=index.tsx) 创建

**Comment by chengzheng777** [comment]
我是苹果电脑，simulator模拟 iphone 15 pro 碰到一样的，请问最后是换了支付宝的my. 吗
